python_modules/dagster/dagster/daemon/daemon.py

```diff
@@ -207,6 +210,10 @@ def _check_add_heartbeat(
                 errors=[error for (error, timestamp) in self._errors],
             )
         )
+        self._num_heartbeats_since_log += 1
+        if self._num_heartbeats_since_log >= LOG_FREQUENCY:
```

Comment:
you could also do this based on a timestamp? right now it's tightly coupled to the heartbeat frequency (which is 30 seconds i think). you could have your own frequency and make it an hour

Reply:
Good idea